### ISSUE TITLE
squiggly heredoc with single quotes indent wrong

### ISSUE THREAD

**Issue by pbl-pw** [jruby/jruby#4260]
### Environment

jruby 9.1.5.0 (2.3.1) 2016-09-23 fffffff Java HotSpot(TM) Client VM 25.51-b03 on 1.8.0_51-b16 +jit [mswin32-x86]

### Test Code
```
obj1 = <<~'VALUE'
line1
line2
	line3
line4
VALUE
puts obj1
obj2 = <<~'VALUE'
	line5
	line6
		line7
	line8
VALUE
puts obj2
```
### Expected Behavior
MRI output:
```
line1
line2
	line3
line4
line5
line6
	line7
line8
```
### Actual Behavior
jruby output:
```
line1
line2
	line3
line4
line5
line6
line7
line8
```

**Comment by andyn95** [comment]
What file is this issue in? Does someone have a link to that file so I can see the code that does this?

**Comment by koic** [comment]
The issue is reproduced in JRuby 9.2.7.

```console
% ruby -v
jruby 9.2.7.0 (2.5.3) 2019-04-09 8a269e3 Java HotSpot(TM) 64-Bit Server VM 25.5-b02 on 1.8.0_05-b13 +jit [darwin-x86_64]
```

I found it when I was working on RuboCop to drop supporting Ruby 2.2.
https://github.com/rubocop-hq/rubocop/pull/7026

**Comment by koic** [comment]
Hi, @headius @enebo. Thank you for the maintenance.
I have encountered this issue repeatedly with RuboCop. Do you have a plan to resolve this issue? I hope this problem is resolved and behaves the same as MRI. I appreciate your activities. Thank you.

**Comment by enebo** [comment]
@koic I will fix this for 9.2.8.0.  I looked briefly and heredoc_indent value is some weird astronomically large value so something is not getting reset or something.

**Comment by koic** [comment]
Thank you very much. I'm looking forward to the release of JRuby 9.2.8.0.
